maps/kleibkhar/kleibkhar.dm

```diff
@@ -1,10 +1,6 @@
 #if !defined(USING_MAP_DATUM)
 	// Mods section
 	#include "../../mods/persistence/_persistence.dme"
-	#include "../../mods/species/ascent/_ascent.dme"
```

Comment:
Uh, we kinda have to test the species? At the very least unathi, and the vore bugs.